### PR TITLE
add AllowedProcMountTypes and ProcMountType to docs

### PR DESCRIPTION
**Pull request by jessfraz**
Docs PR for https://github.com/kubernetes/kubernetes/pull/64283

### CODE REVIEW

File: content/en/docs/concepts/policy/pod-security-policy.md [hunk 1]
```diff
@@ -548,6 +549,21 @@ for the default list of capabilities when using the Docker runtime.
...
+
+`allowedProcMountTypes` is a whitelist of allowed ProcMountTypes.
+Empty or nil indicates that only the `DefaultProcMountType` may be used. 
+
+`DefaultProcMount` uses the container runtime defaults for readonly and masked
+paths for /proc.  Most container runtimes mask certain paths in /proc to avoid
+accidental security exposure of special devices or information. This is denoted
+as the string `Default`.
+
+The only other viable ProcMountType is `UnmaskedProcMount`, which bypasses the
```
Comment (zparnold) [inline content/en/docs/concepts/policy/pod-security-policy.md]: @jessfraz When you say "viable" do you mean that `"Unmasked"` is the only other choice than `"Default"`? If so, you can simply omit the work "viable"

Comment (jessfraz) [reply content/en/docs/concepts/policy/pod-security-policy.md]: updated sorry for the delay!

**Pull request merged**